aten/src/ATen/Context.h

```diff
@@ -119,6 +122,7 @@ class CAFFE2_API Context {
   bool enabled_cudnn = true;
   bool deterministic_cudnn = false;
   bool benchmark_cudnn = false;
+  at::QBackend quantized_engine = at::QBackend::FBGEMM;
```

Comment:
c10/core/Backend already has a backend for quantization,  `Backend::QuantizedCPU`. I think this affects dispatch of quantization related stuff, etc so we don't want to modify this. 
I added `QBackend` to be more of a quantization specific parameter. I agree with renaming `quantized_engine` to `quantized_backend` to make it more consistent (or vice versa - calling it QEngine in case the word backend gets too confusing and overloaded).

Reply:
I think engine might be better, qbackend will be conflicting with current backend name, we want to reduce confusions. 
I think we can do
```
at::QEngine qengine = at::QEngine::FBGEMM;
```
or simply:
```
at::Engine engine = at::Engine::FBGEMM;
```

Reply:
Updated to use QEngine instead of QBackend